.github/pull_request_template.md

```diff
@@ -1,49 +1,53 @@
-<!--
-  Your description goes here: A summary of the change, including any relevant motivation and context.
+## What it does
```

Comment:
I like having a heading for this 👍  I usually add a "background" section before this, because it helps me keep the "description" or "what it does" section short. I'm not sure everyone would use this though.

Reply:
It's not entirely clear to me how this section is different from the general description

Reply:
it's just a header for the general description since we didn't have one before

Reply:
do note that we intentionally removed that header: https://github.com/code-dot-org/code-dot-org/pull/32776

Reply:
I'm having trouble coming up with a compelling practical argument for either side here. Realistically, I think the role of the template is to prompt the PR author to consider including various things in their PR description. While I'd personally prefer to have explicit headings, I don't think including or omitting the "what it does" or "description" header here will affect anything we care about, since people don't usually forget to write a general description. So, either option is fine with me.

Reply:
i like having a heading but i don't feel very strongly, so if we decided to remove it, that's fine with me